specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2019-08-08/storageaccounts.json

```diff
@@ -254,36 +254,28 @@
         "modelAsString": true
       }
     },
-    "WacAccountPermissions": {
-      "description": "Current operation being performed on Storage Account",
+    "StorageAccountKind": {
```

Comment:
Hi. Did you change the existed fields instead of adding new fields.

Reply:
I deleted some existing items which is not needed any more and add some new items.

Reply:
since it's a preview version. It's fine to do this. Because you did many changes, so we need ARM's signoff.